ztunnel-1.24.yaml

```diff
@@ -0,0 +1,78 @@
+package:
+  name: ztunnel-1.24
+  version: 1.24.0
+  epoch: 0
+  description: The `ztunnel` component of istio ambient mesh.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    provides:
+      - ztunnel=${{package.full-version}}
+    runtime:
+      - ca-certificates-bundle
+      - glibc
+      - ip6tables
+      - iptables
+      - libgcc
+      - libmnl
+      - libnetfilter_conntrack
+      - libnfnetlink
+      - wolfi-baselayout
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - cargo-auditable
```

Comment:
OUr pipeline doesn't work here?

Reply:
Unfortunately no, the default build output goes in `out` directory instead of `target/releases` that our pipeline uses.